README.md

```diff
@@ -2,6 +2,12 @@
 
 Simple Node.js package which exposes Swagger docs both in JSON and HTML.
 
+## Installation ##
+
+```
+$ npm install -S @studiohyperdrive/api-docs
```

Comment:
`-S` is overbodig lijkt me, maar mag gerust blijven staan.

Reply:
Hij gaat de package standaard toevoegen als dependency maar is pas vanaf npm 5 dacht ik? Heb Node 6 ook toegevoegd als ondersteunde Node versie en die krijgt standaard npm 3. Kan die engine ook toevoegen in `package.json` maar ik zit wel met mijn gepushte 1.0.1 commit 🙃

Reply:
Alright. Kan blijven staan.